Framework/Editor/V0/AacFlAnimations.cs

```diff
@@ -77,6 +77,53 @@ public AacFlClip BlendShape(SkinnedMeshRenderer[] rendererWithNulls, string blen
             return this;
         }
 
+        public AacFlClip MaterialProperty(Renderer renderer, string propertyName, float value)
+        {
+            MaterialProperty<Renderer>(renderer.transform, $"material.{propertyName}", value);
+
+            return this;
+        }
+
+        public AacFlClip MaterialProperty(ParticleSystemRenderer renderer, string propertyName, float value)
```

Comment:
The method signature should be a `ParticleSystem` type, because a `ParticleSystemRenderer` is already a Renderer.

The aim of exposing ParticleSystem in the signature is that ParticleSystem is itself not a renderer, but it is convenient for the caller to hold the reference to the ParticleSystem object.

The runtime type (generic) remains ParticleSystemRenderer.

Reply:
Will fix